hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/BlockStreamConfig.java

```diff
@@ -44,4 +50,33 @@ public record BlockStreamConfig(
         @ConfigProperty(defaultValue = "32") @NetworkProperty int hashCombineBatchSize,
         @ConfigProperty(defaultValue = "1") @NetworkProperty int roundsPerBlock,
         @ConfigProperty(defaultValue = "localhost") String grpcAddress,
-        @ConfigProperty(defaultValue = "8080") @Min(0) @Max(65535) int grpcPort) {}
+        @ConfigProperty(defaultValue = "8080") @Min(0) @Max(65535) int grpcPort,
+        @ConfigProperty(defaultValue = "3") @NetworkProperty int uploadRetryAttempts,
+        @ConfigProperty(defaultValue = "168") @NetworkProperty int localRetentionHours,
+        @ConfigProperty(defaultValue = "data/config/bucket-credentials.json") @NetworkProperty String credentialsPath,
+
+        // Bucket configurations with default AWS and GCP public buckets
+        @ConfigProperty(
```

Comment:
This looks awkward. Can you please check with @hendrikebbers, if this is the right approach?

Reply:
This is not the correct approach and we should avoid to change the config implementation as it is done in this PR. I forwarded it to @timo0 since he currently extend the config API to be more effective for complexe configurations.

Reply:
Yes, understood! I've synced with him and after the changes he is introducing, we can then write a converter for `CloudBucketConfig` object. I will [link the PR](https://github.com/hashgraph/hedera-services/pull/17088) for better visibility